client/src/components/PrivateRoute.jsx

```diff
@@ -14,18 +14,13 @@ type Props = {
 const PrivateRoute = (props: Props) => {
   const { authed, authRoleIsEquivalentTo } = useAuth();
   const { minRole, path, component } = props;
-
   const equivalent = authRoleIsEquivalentTo(minRole);
 
-  if (authed === true && equivalent === true) {
-    return <Route path={path} exact component={component} />;
-  }
-
-  if (authed === null || equivalent === null) {
-    return null;
-  }
-
-  return <Redirect to="/login" />;
+  return authed && equivalent ? (
```

Comment:
Wondering the same thing here, but it seems this was the fix for the bug so it seems okay

Reply:
Same idea here